.github/workflows/ci-frontend-a11y.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Start API Server for search results
         run: |
           cd ../api
-          make init db-seed-local start &
+          make init db-seed-local populate-search-opportunities start &
```

Comment:
leaving this out was causing failures on the pa11y search page tests, but not sure why it only needs to be added now